tools/packaging/static-build/kernel/build.sh

```diff
@@ -22,8 +22,15 @@ DESTDIR=${DESTDIR:-${PWD}}
 PREFIX=${PREFIX:-/opt/kata}
 container_image="${KERNEL_CONTAINER_BUILDER:-$(get_kernel_image_name)}"
 MEASURED_ROOTFS=${MEASURED_ROOTFS:-no}
+DM_SUPPORT=${DM_SUPPORT:-no}
```

Comment:
Nit: Consistency: we use true/false in `tools/packaging/kernel/build-kernel.sh` rather than yes/no.

Reply:
Fixed.